Herebyfile.mjs

```diff
@@ -404,27 +404,50 @@ export const watchMin = task({
 
 
 
-const { main: lssl, build: buildLssl, watch: watchLssl } = entrypointBuildTask({
+// This is technically not enough to make tsserverlibrary loadable in the
+// browser, but it's unlikely that anyone has actually been doing that.
+const lsslJs = `
+if (typeof module !== "undefined" && module.exports) {
+    module.exports = require("./typescript.js");
+}
+else {
+    throw new Error("tsserverlibrary requires CommonJS; use typescript.js instead");
+}
+`;
+
+const lsslDts = `
+import ts = require("./typescript.js");
+export = ts;
+`;
+
+const lsslDtsInternal = `
+import ts = require("./typescript.internal.js");

```

Comment:
This is a little unfortunate for anyone who is attempting to pull these internal types, e.g. byots/ts-expose-internals but the fix would be quite simple for them; just use `typescript.internal.d.ts` for both.